easybuild/easyconfigs/j/Java/Java-1.8.0_141.eb

```diff
@@ -0,0 +1,20 @@
+name = 'Java'
+version = '1.8.0_141'
```

Comment:
BTW... when I submitted this, 141 was current.   I can't help EB turnaround time  on PRs let another update come out. :)     144 shortly